lib/chartmogul/utils/hash_snake_caser.rb

```diff
@@ -39,12 +39,11 @@ def underscore_key(k)
 
       def underscore(string)
         string.gsub(/::/, '/')
-          .gsub(/([A-Z]+)([A-Z][a-z])/, '\1_\2')
-          .gsub(/([a-z\d])([A-Z])/, '\1_\2')
-          .tr('-', '_')
-          .downcase
+              .gsub(/([A-Z]+)([A-Z][a-z])/, '\1_\2')
```

Comment:
It aligns it on the same column as method call on line 41.
http://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Layout/MultilineMethodCallIndentation